ambry-messageformat/src/test/java/com.github.ambry.messageformat/MessageFormatInputStreamTest.java

```diff
@@ -59,9 +59,9 @@ private void messageFormatBlobPropertiesTest(short blobVersion, BlobType blobTyp
     ByteBufferInputStream stream = new ByteBufferInputStream(ByteBuffer.wrap(data));
 
     MessageFormatInputStream messageFormatStream =
-        (blobVersion == MessageFormatRecord.Blob_Version_V1) ? new PutMessageFormatInputStream(key, prop,
+        (blobVersion == MessageFormatRecord.Blob_Version_V2) ? new PutMessageFormatInputStream(key, prop,
```

Comment:
minor: although the check is the same, this and subsequent ones use the ternary condition, whereas the one in `BlobStoreHardDeleteTest.java` uses regular if-else. Perhaps you could make all these checks into switch-case statements?